server.js

```diff
@@ -109,7 +110,11 @@ function initialiseGlobalMiddleware(app) {
     session({
       store: new RedisStore({ client: redis.client }),
       secret: config.sessionSecret,
-      cookie: { secure: config.https, sameSite: 'lax', maxAge: 24 * 60 * 60 * 1000 }, // 24 hours
```

Comment:
This looks to be the cause of this mornings issue, our sessions were out-living the refresh token lifetime